backend/src/main/java/com/crm/service/impl/CarServiceImpl.java

```diff
@@ -39,6 +40,14 @@
 
     @Override
     public CarResponse getCarById(final Long id) {
-        return carMapper.convertToDto(carRepository.findById(id).orElseThrow(() -> new CarNotFoundException(ErrorDict.CAR_NOT_FOUND)));
+        return carMapper.convertToDto(carRepository.findById(id)
+                .orElseThrow(() -> new CarNotFoundException(ErrorDict.CAR_NOT_FOUND)));
```

Comment:
code lines formatting

Reply:
Changed to what was before. Is that what you meant?